test/system/list:imported.test.ts

```diff
@@ -15,7 +15,7 @@ describe('`snyk-api-import list:imported <...>`', () => {
     process.env = { ...OLD_ENV };
   });
   it('Shows help text as expected', (done) => {
-    return exec(`node ${main} list:imported help`, (err, stdout) => {
+    exec(`node ${main} list:imported help`, (err, stdout) => {
```

Comment:
dropping returns as later version of `jest` now yells more loudly to either use `done()` or return a promise.